torch/onnx/symbolic_opset9.py

```diff
@@ -4752,7 +4759,7 @@ def _pack_padded_sequence(g: jit_utils.GraphContext, input, lengths, batch_first
     # It's really only necessary because those operators expand to something that
     # only works with int32 types in Caffe2...
     if lengths.type().scalarType() != "Int":
-        lengths = _cast_Int(g, lengths, False)  # type: ignore[name-defined]
+        lengths = g.op("Cast", lengths, to_i=_C_onnx.TensorProtoDataType.INT32)
```

Comment:
`_C_onnx.TensorProtoDataType.INT32`

I'm starting to think if we should have wrapper classes to hide away the `_C.onnx`, and likewise `_C.torch` details.

Reply:
Maybe replace everything with JitScalarType then?